include/Request/LoadTaxonomyTerm.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Custom Taxonomy select2 Ajax Action Class.
+ *
+ * @since 2.0.0
+ */
+class BM_Request_LoadTaxonomyTerm {
+
+	/**
+	 * Entry point for the BM_Request_LoadTaxonomyTerm class.
+	 *
+	 * @since 2.0.0
+	 */
+	public function load() {
+		add_action( 'wp_ajax_bm_load_taxonomy_term', array( $this, 'bm_load_taxonomy_term' ) );
+		add_filter( 'bm_javascript_array', array( $this, 'include_ajax_params_in_localization' ) );
+	}
+
+	/**
+	 * Ajax call back function for getting taxonomies to load select2 options.
+	 *
+	 * @since 2.0.0
+	 */
+	public function bm_load_taxonomy_term(){
+		check_ajax_referer( 'bulk-move-posts', 'nonce' );
+		$return = array();
+
+		$terms = get_terms( array(
+			'taxonomy'   => sanitize_text_field($_GET['term']),
```

Comment:
Run `phpcs` and fix all issues in this file

Reply:
@rajanit2000 This comment is not addressed.

Reply:
@sudar I have to install phpcs package on my sublime text 3, But it does not show any warnings

Reply:
You can quickly test it by running the command `vendor/bin/phpcs` from root directory.

Some example of code formatting issues are

`sanitize_text_field($_GET['term']),` -> Doesn't have proper space.
`bm_load_taxonomy_term(){` -> Space needed before `{`

Reply:
@rajanit2000 

This is not fixed yet.

Reply:
@sudar you only say revert the `phpcs` changes. We can update it once we have done all things.